Strings/Items/Father's Blessing_short.md

```diff
@@ -0,0 +1,13 @@
+# Father's Blessing (with shorter description)
+
+| Language | Description |
+| :------- | :---------- |
+| Text ID | 3635 |
+|||
+| English | <h3>**Father's Blessing**</h3>Using this item will activate the blessing.<br><br>With the blessing active, every mist raven<br>occurance will summon Owl briefly as a memory.<br>Costs 2 spirit emblems per cast.<br><br>To disable the blessing, rest at a Sculptor's Idol.<h3> |
+|||
+| Japanese | <h3>**義父の加護**</h3>使えば、義父の加護をその身に受けられる<br><br>加護を受けているときに「霧がらす」を使用すると、<br>形代を2つ、追加で消費する代わりに、<br>短時間だけ義父のまぼろしと共闘できる<br><br>ただし、鬼仏で休息すると加護は消えてしまう<h3> |
+|||
+| Korean | <h3>**의부의 축복**</h3>Translation Needed<h3> |
```

Comment:
What is the purpose of h3 tags here that is not used in Strings/EventText/EventText.md and other places